### PR TITLE
fix(task details):task details bug

### PR DESCRIPTION
**Pull request by Bankar-Swapnil**
### Functionality:
fix comment section and log-work functionality
### Solution:
Changes done in task-details component 

### Risk level:
- [ ] high 
- [ ] medium
- [x] low

### How to test:
go to task detail page and comment and log-work and mark task completed without marking log-work 100% and submit it automatically mark completed and on reopen it mark 0%.

### CODE REVIEW

**Review by Pendyala-thanvi — APPROVED** [review]

File: src/app/body/task-details/log-work/log-work.component.html [hunk 1]
```diff
@@ -13,28 +13,30 @@
...
-                </div>
-                <div class="col-2 col-md-2">
-                    <button type="button" class="close col-md" data-dismiss="modal" (click)="workDone()" aria-label="Close">
-                        <span aria-hidden="true" class="material-icons">close</span>
-                    </button>
+        <div class="modal-content">
+            <div class="modal-body pb-0">
+                <div class="row">
+                    <div class="col-md-10 col-10">
+                        <h2>Create New Task</h2>
```
Comment (viv9k) [inline src/app/body/task-details/log-work/log-work.component.html:20]: Create new task for log work???

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required fix

File: src/app/body/task-details/log-work/log-work.component.html (hunk 1 above)
Comment (Bankar-Swapnil) [reply src/app/body/task-details/log-work/log-work.component.html]: Changes Done

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**